core/schema/module.go

```diff
@@ -1186,12 +1206,6 @@ func (s *moduleSchema) moduleSource(ctx context.Context, query *core.Query, args
 		src.AsGitSource.Value.Version = gitCommit // TODO preserve semver here
 		src.AsGitSource.Value.Commit = gitCommit  // but tell the truth here
```

Comment:
Oh hey, I think this is the point where we can actually address this TODO which has been carried around for eons. :joy: 

Can we set `.Version` to the original `modVersion` i.e. the one that was `isSemver()`? Tried to suggest changes, but we reassign the var. Couild maybe assign a new `commit` var instead.

Normally wouldn't block on addressing an extra TODO, but we're about to start using this `Version` value in analytics, so it's a good time to address it.